examples/ITR_UI.py

```diff
@@ -608,12 +619,13 @@ def update_graph(
     fig1.update_layout(legend=dict(orientation="h", yanchor="bottom", y=1, xanchor="center", x=0.5))
 
     # Covered companies analysis
-    coverage = filt_df[['company_id', 'ghg_s1s2', 'cumulative_target']].copy()
+    scope_column = 'ghg_' + EI_bm.scope_to_calc.name.lower()
+    coverage = filt_df[['company_id', scope_column, 'cumulative_target']].copy()
     zeroE = Q_(0, 't CO2')
```

Comment:
Note that in the following lines, the EXISTENCE of GHG_SCOPE12 is tested (which is value) but not the value (other than NaN, zero, or non-zero).  Per earlier comments, that's fine--this is a way of telling whether data in a row exists.  But any use of GHG_SCOPE12 as a value for actual weighting or computations would be highly suspect.

Reply:
We clarified that this use of GHG_SCOPE12 is valid, but any use other than initializing projections is not really value (such as for a weighting).

Reply:
@MichaelTiemannOSC 
i found this article (https://sciencebasedtargets.org/resources/legacy/2020/09/Temperature-Rating-Methodology-V1.pdf)
it suggests (page 22), that GHG weights only apply only for S1S2S3.
the table 5 shows, that S1S2 goes w/o weights, and S3 goes w/o weights.

Are you sure we need to apply weights for this PR?
Looks like it's for S1S2S3 only

Reply:
I additionally looked through the os-climate internal document "BUILDING OPEN SOURCE PORTFOLIO ALIGNMENT CAPABILITIES
METHODOLOGY & PROCESS DOCUMENTATION".

On page 15, it says:

> Temperature weighting: The two separately calculated temperature scores can be transparently combined using a (potentially) company-specific probability measure p(c) for the estimated likelihood that the target will be reached:
𝐼𝑇𝑅(𝑐) = 𝑝(𝑐) ∗ 𝐼𝑇𝑅(𝑐, 𝑡𝑎𝑟𝑔𝑒𝑡) + (1 – 𝑝(𝑐)) ∗ 𝐼𝑇𝑅(𝑐, 𝑡𝑟𝑎𝑗𝑒𝑐𝑡𝑜𝑟𝑦)

AFAIU, it confirms, that weights should apply to combined score S1S2S3, but not to individual scored S1S2 and S3.

Reply:
Based on 2 references i posted earlier, i removed the commit, which earlier introduced weights to S1S2 and S3.
I think, there was a misunderstanding of the methodology, which from my perspective only recommends weights for combined score S1S2S3.
I understand that you are sure, that weight need to apply any way. If you still think that's the case, please reference some source of methodology, which convinces you in necessity of such move.

I left the adapted unit tests, so, we now have a better test coverage of S3 calculations.

Sanity check: for the company CBRE with high S3 emissions, the notebook `s1s2_s3_calc.ipynb` now calculates high S3 temperature score, as we might expect.